src/css/Validation.js

```diff
@@ -1,7 +1,11 @@
-//This file will likely change a lot! Very experimental!
-/*global Matcher, Properties, ValidationTypes, ValidationError, PropertyValueIterator */
-/*exported Validation */
-var Validation = {
+"use strict";
+var Matcher = require("./Matcher");
+var Properties = require("./Properties");
+var ValidationTypes = require("./ValidationTypes");
+var ValidationError = require("./ValidationError");
+var PropertyValueIterator = require("./PropertyValueIterator");
+
+var Validation = module.exports = { /* exported Validation */
```

Comment:
There are two patterns we need to use to make these into proper node modules.  The majority of modules export a function.  In that case, the pattern is:
```
"use strict";

module.exports = <FunctionName>;

...imports...

function <FunctionName>(....) {
```

But for three files (Validation, Properties, and Colors) the thing exported is not a function, so we need a slightly different pattern:
```
"use strict";

...imports...

var <ObjectName> = module.exports = { /* exported <ObjectName> */
```
The `/* exported */` is necessary because the module may not actually refer to `<ObjectName>` anywhere in the file, so you need the comment to tell `jshint` that it's okay that it appears unused.  I decided to put it on the same line as `module.exports` to keep the pattern together.

I could move it if you feel strongly.  I guess that makes the second line in the file always describe the thing being exported, either via `module.exports = <FunctionName>` or by `/* exported <ObjectName> */`.

In the longer term, it would be nice if the exported thing were always a Function, for consistency.